### ISSUE TITLE
research: is it possible to run PowerBi in a Dockercontainer

### ISSUE THREAD

**Issue by enningb** [ed2c/docker_1cho#5]
It would be nice if we can run PowerBI in a Dockercontainer. But can windows be containerized?

**Comment by Tomeriko96** [comment]
Below follow some links to examples hosted on GitHub

https://github.com/HealisticEngineer/Docker-PowerBI

https://github.com/attilaersek/powerbi-php-sample/blob/master/docker-compose.yml

https://github.com/InternationalTradeAdministration/ita-powerbi-gateway/blob/master/docker-compose.yml

**Comment by CorneeldH** [comment]
@Tomeriko96 It would be very cool if we could give this a shot before the 24th..

**Comment by CorneeldH** [comment]
![image](https://github.com/ed2c/docker_1cho/assets/6919390/cdf23ca5-6cbb-4999-b906-193e8f5fa069)

en laatste linkje: https://www.cloudly.today/power-bi/power-bi-for-mac-a-comprehensive-guide/

**Comment by Tomeriko96** [comment]
We kunnen een windowsVM opzetten, daarbinnen kan je dan docker desktop gebruiken. Een eigen docker voor powerbi zelf heb ik vanochtend weer gezocht, maar lijkt simpelweg (nog) niet ondersteund

**Comment by CorneeldH** [comment]
ja, laten we powerbi lekker zitten toch, met huidige ontwikkelingen is dat alleen maar een drag.Ik gooi deze helemaal dicht
